src/commands/PullRequest/FilterLabels.ts

```diff
@@ -0,0 +1,56 @@
+import debug from 'debug';
+import { Arguments, Argv, CommandModule } from 'yargs';
+
+/**
+ * Nuxeo Preset Purge Command - Purge previously installed Nuxeo Preset
+ */
+export class FilterLabels implements CommandModule {
+  public command: string = 'filter-labels';
+
+  public describe: string = 'Filter the labels for the given mode (test or preview)';
+
+  private readonly log: debug.IDebugger = debug('command:pr:filter-labels');
+
+  public builder: (args: Argv) => Argv = (args: Argv) => {
+    args.options({
+      mode: {
+        alias: ['m'],
+        description: 'The type of labels to filter, either `test` or `preview`',
+        required: 'true',
+        default: 'preview',
```

Comment:
Do we want to have those modes absolutely or can be without mode (hence all of them) to make them work with both test and previews to avoid redundancy?